workflow/scripts/simulate_reads.py

```
@@ -0,0 +1,30 @@
+from os import path
+from snakemake.shell import shell
+
+# define default options
+read_length = snakemake.params.get("read_length", 70)
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)